packages/autocomplete-js/src/__tests__/translations.test.ts

```diff
@@ -0,0 +1,111 @@
+import { fireEvent, waitFor } from '@testing-library/dom';
+
+import { autocomplete } from '../autocomplete';
+
+describe('translations', () => {
```

Comment:
We don't test `submitButtonTitle` here. We could have two `describe`s: one for regular mode, one for detached mode that overrides `matchMedia`.